packages/components/src/components/search/_search.scss

```diff
@@ -51,7 +51,7 @@
     }
 
     &::placeholder {
-      color: $text-05;
+      @include placeholder-colors;
```

Comment:
Search should be using `text-05` since placeholder text is being changed back to `text-03`